docs/src/pages/demos/tooltips/tooltips.md

```diff
@@ -52,12 +52,12 @@ By default disabled elements like `Button` do not trigger user interactions so a
 
 {{"demo": "pages/demos/tooltips/DisabledTooltips.js"}}
 
+## Customized Tooltips
+
+{{"demo": "pages/demos/tooltips/CustomizedTooltips.js"}}
+
```

Comment:
@mbrookes Trust I'm very hesitant to have this property. From my point of view, the main value of this pull requests comes from the default value it's setting: `max-width: 300`.
Regarding having a dedicated property for that? Well, why not using CSS override?
I have the intuition that people will be more interested in the `Customized Tooltips` tooltip demo than in the max-width. Benchmarking the other tooltip implementations, I have seen very few having this type of customization. Anyway. I want to add analytics on the live documetation regarding demos usage. I think that we should sort the demos, when possible by their popularity, so overhall, people scroll less in the pages to find what they are looking for.

Reply:
@oliviertassinari If it helps I always scroll to the bottom and love viewing all the demos down to the last one :). Another way to know is to gauge the number of user questions that can be associated with not seeing the demos at the bottom. Do you ever have to say "Scroll down to the last demo example?" and how often. 

I also inserted the demo example before the last one on purpose, because it pertains to a standard property. It seemed odd to talk about css customization then document a basic property last. This is a shift from the pattern I'm used to see elsewhere.

Reply:
@simoami Oh, Interesting habit, I'm not the only one then!